corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -135,6 +135,7 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
                     title: menuResponse.title,
                     description: menuResponse.description,
                     sidebarEnabled: true,
+                    disableDynamicSearch: sessionStorage.submitPerformed === undefined ? true : !sessionStorage.submitPerformed,
```

Comment:
Isn't this equivalent to this?
```suggestion
                    disableDynamicSearch: !sessionStorage.submitPerformed,
```